### PR TITLE
Merqure.rbmq.metadata

### PR DESCRIPTION
**Pull request by BriceFrancois**
- HeaderProperties' values as object
- Message Priority Support
- DeadLetter Policy Support
- Exchange type support: direct, fanout, topic, headers

### CODE REVIEW

**Comment by BriceFrancois** [comment]
:white_check_mark: [Build MerQure 1.0.44 completed](https://ci.appveyor.com/project/BriceFrancois/merqure/build/1.0.44) (commit https://github.com/LuccaSA/MerQure/commit/75b2515c68 by @BriceFrancois)

**Comment by BriceFrancois** [comment]
:white_check_mark: [Build MerQure 1.0.46 completed](https://ci.appveyor.com/project/BriceFrancois/merqure/build/1.0.46) (commit https://github.com/LuccaSA/MerQure/commit/9a5b1770d8 by @BriceFrancois)

File: src/MerQure/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MerQure
```
Comment (thomaf) [inline src/MerQure/Constants.cs]: A voir, mais il faut peut-être mettre ces constantes dans RbMQ...

Comment (BriceFrancois) [inline src/MerQure/Constants.cs]: c'est noté

**Comment by BriceFrancois** [comment]
:white_check_mark: [Build MerQure 1.0.48 completed](https://ci.appveyor.com/project/BriceFrancois/merqure/build/1.0.48) (commit https://github.com/LuccaSA/MerQure/commit/e8df26ab26 by @BriceFrancois)

**Comment by BriceFrancois** [comment]
:white_check_mark: [Build MerQure 1.0.50 completed](https://ci.appveyor.com/project/BriceFrancois/merqure/build/1.0.50) (commit https://github.com/LuccaSA/MerQure/commit/45756ac60f by @BriceFrancois)

**Pull request merged**